index.js

```diff
@@ -86,8 +88,55 @@ module.exports = postcss.plugin('postcss-bem-linter', function(primaryOptions, s
       });
     }
 
+    function checkGlob(file, globs) {
+      // PostCSS turns relative paths into absolute paths
+      file = path.relative(process.cwd(), file);
+      return minimatchList(file, globs);
+    }
+
+    function isImplicitComponent(file) {
+      if (Array.isArray(config.implicitComponents)) {
```

Comment:
I believe `Array.isArray` is fully supported in `0.12.x`. The `util.isArray` even proxies calls to it.

See here - https://github.com/nodejs/node/blob/v0.12.0-release/lib/util.js#L512

I think we can revert https://github.com/postcss/postcss-bem-linter/pull/93/commits/4a0c46db2f85f61629a9133fe1684b811bdd31b5 as it's more reliable than `instanceof` afaik.